### PR TITLE
Update cli-v2-install.md

### PR DESCRIPTION
**Pull request by mxplusb**
WSL > Linux VM.

### CODE REVIEW

File: content/cli-v2-install.md [hunk 1]
```diff
@@ -109,7 +109,7 @@ $ gem install yajl-ruby
 
 ### Windows
 
-The `create-env` and `destroy-env` commands are not yet supported on Windows. Feel free to give it a try (and let us know if you have feedback), but we would recommend using a Linux-based virtual machine to run those commands instead.
+The `create-env` and `destroy-env` commands are not yet supported on native Windows. Feel free to give it a try (and let us know if you have feedback), but we would recommend levering the Windows Subsystem for Linux if you need to run either command.
```
Comment (dpb587-pivotal) [inline content/cli-v2-install.md]: "levering" » "leveraging"? unless it's Windows terminology I'm not familiar with?

Comment (mxplusb) [reply content/cli-v2-install.md]: Nope, totally had a typo. Thanks!

**Comment by mxplusb** [comment]
fixed typo in review.

**Comment by dpb587-pivotal** [comment]
@mxplusb thinking more about this, have you tested `create-env` and `delete-env` with the Linux subsystem on Windows? I don't think it would actually work, so am having second thoughts on this change.

**Comment by dpb587-pivotal** [comment]
Interesting, and good to know. I'll go ahead and merge then.

Windows support is not currently a priority, but it's great to hear it has been working for you. I know @freddesbiens has been considering more of the Windows use cases. He may be interested in following up with you at some point to hear more about your experiences and feedback.

**Comment by mxplusb** [comment]
@dpb587-pivotal I'm a Windows user by default with extensive Windows knowledge, so please let me know if you need help/assistance with Windows-based activities and I can work with you to get it sorted. :)

**Comment by freddesbiens** [comment]
Glad to hear that! We will have a discovery activity around this; probably
early this fall. I will probably reach out at that time.

Thank you for your involvement in our community!

Best,

    Frédéric Desbiens
    Product Manager | Cloud Foundry BOSH

**Pull request merged**